piezo_web_app/PiezoWebApp/tests/services/spark_job/spark_ui_service_test.py

```diff
@@ -0,0 +1,88 @@
+import kubernetes
+import mock
+import pytest
+
+from PiezoWebApp.src.services.spark_job.spark_ui_service import SparkUiService
+from PiezoWebApp.src.utils.configurations import Configuration
+
+
+class TestSparkUiService:
```

Comment:
There are no unhappy test cases here: what happens if e.g. `job_name` is `Null`?